tests/test_utils/graphs.py

```diff
@@ -111,20 +111,27 @@ def example_hin_1_nx(feature_name=None, for_nodes=None, feature_sizes=None):
 
 
 def example_hin_1(
-    feature_sizes=None, is_directed=False, self_loop=False
+    feature_sizes=None,
+    is_directed=False,
+    self_loop=False,
+    node_types=("A", "B"),
```

Comment:
Thoughts on not exposing `node_types` and `edge_types` like this and just update the tests to use "A", "B", etc? I don't really see a particular use-case for any new tests wanting to use different names for the types for the same underlying graph, and the existing test probably doesn't need the `user` and `movie` context?